src/css/style.css

```diff
@@ -7,7 +7,7 @@ body {
   background-image: url("../img/mainbg.jpg");
   background-repeat: no-repeat;
   background-size: 100% 100%;
-  height: auto;
+  height: 664px;
```

Comment:
- [ ] On the last line in the stylesheet there is an `!important` tag. Kindly remove it avoiding CSS bad practices.